app-backend/db/src/main/scala/com/azavea/rf/database/TemplateTagDao.scala

```diff
@@ -0,0 +1,68 @@
+package com.azavea.rf.database
+
+import java.sql.Timestamp
+
+import com.azavea.rf.database.Implicits._
+import com.azavea.rf.datamodel._
+import doobie._
+import doobie.implicits._
+import doobie.postgres._
+import doobie.postgres.implicits._
+import cats._
+import cats.data._
+import cats.effect.IO
+import cats.implicits._
+import java.util.UUID
+
+object TemplateTagDao extends Dao[TemplateTag] {
+  val tableName = "template_tags"
+
+  val selectF = sql"""
+    SELECT template_id, tag_id
+    FROM
+    """ ++ tableF
+
+  def insert(template: Template, tag: Tag): ConnectionIO[TemplateTag] = {
+    sql"""
+    INSERT INTO template_tags (template_id, tag_id)
+    VALUES (${template.id}, ${tag.id})
+    """.update.withUniqueGeneratedKeys[TemplateTag]("template_id", "tag_id")
+  }
+
+  def delete(template: Template, tag: Tag): ConnectionIO[Int] = {
+    query.filter(fr"template_id = ${template.id}")
+      .filter(fr"tag-id = ${tag.id}")
+      .delete
+  }
+
+  def createMany(template: Template, tagList: NonEmptyList[UUID]): ConnectionIO[Int] = {
+    val insertQuery = "INSERT INTO template_tags (template_id, tag_id) VALUES (?, ?)"
+    Update[(UUID, UUID)](insertQuery).updateMany(tagList.map(tagId => (template.id, tagId)))
+  }
+
+  def setTemplateTags(template: Template, tagList: List[UUID]): ConnectionIO[Int] = {
```

Comment:
We called an analogous method [`replaceScenesInProject`](https://github.com/raster-foundry/raster-foundry/blob/7302562c249efcc54902732817f3d03892ac94e7/app-backend/db/src/main/scala/com/azavea/rf/database/ProjectDao.scala#L259-L272) in the `ProjectDao`, which I think was more transparent